sdk/formrecognizer/azure-ai-formrecognizer/src/samples/java/com/azure/ai/formrecognizer/FormRecognizerClientJavaDocCodeSnippets.java

```diff
@@ -250,10 +249,10 @@ public void beginRecognizeReceiptsFromUrl() {
      */
     public void beginRecognizeReceipts() throws IOException {
```

Comment:
is there a reason why we left this code?

Reply:
Oh yeah, receipt design changes are part of a separate PR https://github.com/Azure/azure-sdk-for-java/pull/12380. Haven't merged that one yet!